src/dhttest/cases/neo/GetAll.d

```diff
@@ -164,6 +164,80 @@ public class GetAllSuspend : NeoDhtTestCase
 
 /*******************************************************************************
 
+    Test case which starts a GetAll then kills all connections and checks that
+    the GetAll request successfully reconnects and continues.
+
+*******************************************************************************/
+
+public class GetAllConnError : NeoDhtTestCase
+{
+    import turtle.env.ControlSocket : sendCommand;
+    import ocean.task.Task;
+    import ocean.io.select.client.TimerEvent;
+    import ocean.math.random.Random;
+    import swarm.neo.client.requests.NotificationFormatter;
+
+    override public Description description ( )
+    {
+        Description desc;
+        desc.name = "Neo Puts followed by neo GetAll with connection drop";
+        return desc;
+    }
+
+    public override void run ( )
+    {
+        auto task = Task.getThis();
+        auto rand = new Random;
+
+        const num_records = 1000;
+        ubyte[] val;
+        val.length = 8 * 1024;
+        for ( hash_t key = 0; key < num_records; key++ )
+        {
+            foreach ( ref b; val )
+                b = rand.uniform!(ubyte)();
+
+            auto res = this.dht.blocking.put(this.test_channel, key, val);
+            test(res.succeeded);
+        }
```

Comment:
Use the function you just added in the previous commit ;-)

Reply:
`putRecords`. Unless it's GitHub's wrong commit order thing?

Reply:
I think you're reading backwards.